src/Fable.Elmish.React.fsproj

```diff
@@ -13,6 +13,6 @@
   <ItemGroup>
     <PackageReference Update="FSharp.Core" Version="4.7.*" />
     <PackageReference Include="Fable.Elmish" Version="4.0.0-beta-*" />
-    <PackageReference Include="Fable.React" Version="6.*" />
+    <PackageReference Include="Fable.React" Version="8.*" />
```

Comment:
Are those bindings backwards-compatible? Or are we forcing elmish users to upgrade to React18?
Conversely, can we keep referencing v6 bindings, but let the app developer redirect/overwrite to v8 bindings?

Reply:
Don't know about going from 6 to 7, but 7 to 8 should be safe, it's [just additions](https://github.com/fable-compiler/fable-react/pull/226).

> Conversely, can we keep referencing v6 bindings, but let the app developer redirect/overwrite to v8 bindings?

I don't see how it would be feasible to use the new API here then.